src/main/resources/updater/Instruction to use past versions.txt

```diff
@@ -0,0 +1 @@
+Copy the relevant version JAR to the root folder (no need to rename it) and run that JAR file.
```

Comment:
Is this instruction to use backup versions enough? What happens is all backup version will be in `past_versions/`. This `instruction.txt` will be in that folder as well.

To use a backup version, user just need to copy that JAR backup to the root and run that file.